src/main/scala/chisel3/util/Decoupled.scala

```diff
@@ -88,6 +88,8 @@ class QueueIO[T <: Data](gen: T, entries: Int) extends Bundle
   val deq   = Decoupled(gen.cloneType)
   /** The current amount of data in the queue */
   val count = UInt(OUTPUT, log2Up(entries + 1))
+
+  override def cloneType: this.type = { new QueueIO(gen, entries).asInstanceOf[this.type]; }
```

Comment:
The ; and {} are unnecessary and do not follow the style of other cloneType methods.